rest-ehr-scape/src/main/java/org/ehrbase/rest/ehrscape/controller/EhrController.java

```diff
@@ -136,11 +138,27 @@ public ResponseEntity<EhrResponseData> updateStatus(@PathVariable("uuid") UUID e
 
   private EhrStatus extractEhrStatus(@RequestBody String content) {
     EhrStatus ehrStatus = new EhrStatus();
+    ehrStatus.setArchetypeNodeId("openEHR-EHR-EHR_STATUS.generic.v1");
+    ehrStatus.setName(new DvText("EHR Status"));
 
     if (StringUtils.isNotBlank(content)) {
       Gson json = new GsonBuilder().create();
       Map<String, Object> atributes = json.fromJson(content, Map.class);
 
+      Optional<String> subjectId =
+          Optional.ofNullable(atributes.get("subjectId"))
+              .map(String.class::cast)
+              .filter(StringUtils::isNotBlank);
+      Optional<String> subjectNamespace =
+          Optional.ofNullable(atributes.get("subjectNamespace"))
+              .map(String.class::cast)
+              .filter(StringUtils::isNotBlank);
+      if (subjectId.isEmpty() || subjectNamespace.isEmpty()) {
+        throw new InvalidApiParameterException("subjectId or subjectNamespace missing");
+      }
+      PartySelf subject = new PartySelf(new PartyRef(new HierObjectId(subjectId.get()), subjectNamespace.get(), "PERSON"));
+      ehrStatus.setSubject(subject);
+
       if (atributes.containsKey(MODIFIABLE)) {
         ehrStatus.setModifiable((Boolean) atributes.get(MODIFIABLE));
       }
```

Comment:
I think quarriable is missing here

Reply:
It is there, but hidden in the diff view because it did not change. Line 148-150 ;)